src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -59,6 +59,14 @@ class DialogmoteService(
         }
     }
 
+    fun getDialogmoteUnfinishedList(enhetNr: EnhetNr): List<Dialogmote> {
+        return database.getDialogmoteList(enhetNr)
```

Comment:
Her blir det jo tusenvis av møter man må iterere igjennom etter hvert, så jeg synes det hadde vært litt bedre å gjøre filtreringen i db-queriet.

Nå som det ikke er så mange møter i databasen går det sikkert fint, men vi kommer vel ikke til å slette møter noen gang.

Reply:
Ja, enig i at det hadde vært bedre, kan gjøre et nytt forsøk på det.
Skal vel holde med å tilpasse til `WHERE tildelt_enhet = ? AND status IN ('INNKALT', 'NYTT_TID_STED')` i sql og en ny indeks på `mote (tildelt_enhet, status)` ?

Reply:
Ja, det høres bra ut (optimalt med den indeksen du foreslår, men funker antakelig ganske bra uten også) 👍

Reply:
Fikset!